war/src/main/less/base/style.less

```diff
@@ -1327,6 +1384,18 @@ DIV.to-be-removed { display: none; }
 
 /* ========================= Other form related CSS ========================= */
 
+.form-container {
+  max-width: 100%;
+}
+
+.optionalBlock-container > .form-container,
+.radioBlock-container > .form-container,
+.dropdownList-container {
```

Comment:
This one is not looking good for `.dropdownList-container`
<img width="848" alt="Captura de pantalla 2020-06-26 a las 10 20 55" src="https://user-images.githubusercontent.com/5738588/85836502-f31d9100-b796-11ea-9738-9dd5ae445573.png">